js/web/web-backend.js

```diff
@@ -0,0 +1,164 @@
+import plonkWasm from '../../../web_bindings/plonk_wasm.js';
```

Comment:
`js/web` moved here from `compiled/web_bindings`.
TODO: refactor this to TypeScript